src/com/trollworks/gcs/spell/Spell.java

```diff
@@ -142,8 +152,7 @@
     private String                 mCastingTime;
     private String                 mDuration;
     private int                    mPoints;
-    private int                    mLevel;
-    private int                    mRelativeLevel;
+    private SkillLevel             mLevel;
```

Comment:
I don't see where you are guaranteeing that `mLevel` is properly allocated in all the places where you done this sort of change. This wasn't an issue with the old code, as primitives were used. The old code is also generally faster, as it doesn't have to go through a function hop -- especially with skill/spell levels, that can be important, as they are touched a lot, especially on sheets that have a lot of them.

Reply:
Note: I'm not against the change to use the SkillLevel class for these two bits of data, assuming the code guarantees the object is always allocated and performance doesn't noticeably drop on sheets that have hundreds of skills & spells on them.